projects/bsquared/index.js

```diff
@@ -37,8 +13,8 @@ module.exports = {
   methodology: "Staking tokens via BSquared Network Buzz counts as TVL",
   bitcoin: {
     tvl: sdk.util.sumChainTvls([
-      sumTokensExport({ owners: BTCOwners }),
-      sumBRC20TokensExport({ owners: BRC20Owners }),
+      sumTokensExport({ owners: bitcoinAddressBook.bsquared.BTC }),
```

Comment:
instead of nested, we can keep the keys as 'bsquared_BTC' & 'bsquared_BRC20', this way we have a flat structure